src/views/layouts/main.hbs

```diff
@@ -1,15 +1,66 @@
-<html lang="en">
+<html lang="en" dir="ltr">
   <head>
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
     />
-    <title></title>
+    <title>
+      {{#if title}}
+        {{title}} - Frontend Forward 2024
+      {{^}}
+        Frontend Forward 2024
+      {{/if}}
+    </title>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="view-transition" content="same-origin" />
     <script src="https://unpkg.com/htmx.org@2.0.0"></script>
+    <style>
+      @view-transition {
+        navigation: auto;
+      }
+      @keyframes collapse {
+        from {
+          transform: rotate(0) scale(1) translateX(0) translateY(0);
+        }
+        to {
+          transform: rotate(-300deg) scale(0) translateX(-100%) translateY(100%);
+        }
+      }
+      @keyframes expand {
+        from {
+          clip-path: inset(50% 50% 50% 50%);
+          transform: scale(0) translateX(100%) translateY(-70%);
+          background: skyblue;
+        }
+        to {
+          clip-path: inset(0 0 0 0);
+          transform: scale(1) translateX(0) translateY(0);
+          background: inherit;
+        }
+      }
+      ::view-transition-old(page-transition) {
+        animation: 600ms ease-out both collapse;
+      }
+      ::view-transition-new(page-transition) {
+        animation: 600ms ease-in both expand;
+      }
+      body {
+        view-transition-name: page-transition;
+      }
+    </style>
```

Comment:
It's not for me but it might be jittery in general. I just wanted to put some animation here for the time being. If you have a suggestion for what we should have instead, I can change it! Or we can change it in another PR too. I'm not super sold on the animation itself, I just wanted to have something in place 🙂